### PR TITLE
lint: set style for variable name

### LINKED ISSUE #1628: maintenance: consistency in variables names
Another maintenance idea:
One naming convention in variables names.

It may be pretty easy to execute since in #1615 I have added plugin for it.
Options are:
```
      { "type": "default", "format": "", "leadingUnderscore": "allow", "trailingUnderscore": "allow" },
      { "type": "variable", "modifiers": ["global", "const"], "format": "" },
      { "type": "variable", "modifiers": ["export", "const"], "format": "" },
      { "type": "functionVariable", "modifiers": ["export", "const"], "format": "" },
      { "type": "parameter", "modifiers": "unused", "leadingUnderscore": "allow" },
      { "type": "member", "modifiers": "private", "leadingUnderscore": "allow" },
      { "type": "member", "modifiers": "protected", "leadingUnderscore": "allow" },
      { "type": "method", "filter": "^toJSON$", "format": null },
      { "type": "property", "modifiers": ["public", "static", "const"], "format": "" },
      { "type": "type", "format": "PascalCase" },
      { "type": "class", "modifiers": "abstract", "prefix": "" },
      { "type": "interface", "prefix": "" },
      { "type": "genericTypeParameter", "prefix": "" },
      { "type": "enumMember", "format": "PascalCase" }
```

More info: https://github.com/ajafff/tslint-consistent-codestyle/blob/master/docs/naming-convention.md

### PR DESCRIPTION
**Pull request by kmdrGroch**
fixes #1628

### CODE REVIEW

**Comment by kmdrGroch** [comment]
finally, I have done it. My fingers hurt :x

**Comment by kmdrGroch** [comment]
Strange, my VSC said there was no error... I'll fix it today or tomorrow

File: packages/core/test/unit/Sandbox.spec.ts [hunk 1]
```diff
@@ -190,8 +190,8 @@ describe(Sandbox.name, () => {
         '1',
         mutant,
         new SourceFile(new File('foobar.js', 'original code')));
-      testableMutant.selectTest(testResult({ timeSpentMs: 10 }), 1);
-      testableMutant.selectTest(testResult({ timeSpentMs: 2 }), 2);
+      testableMutant.selectTest(TEST_RESULT({ timeSpentMs: 10 }), 1);
```
Comment (nicojs) [inline packages/core/test/unit/Sandbox.spec.ts]: As stated earlier, I think we should leave the casing of functions camelCase

**Comment by kmdrGroch** [comment]
Nico I can see why it's crashing. You can see what has been done by now so I would like you to add some overall comment about any `style changes`.

**Comment by nicojs** [comment]
I've added some comments. :)

**Comment by kmdrGroch** [comment]
So basically you want all `exported primitives` to be UPPER_CASE, and `exported fuctions and objects` to be camelCase?

**Comment by nicojs** [comment]
Great :+1:

**Comment by kmdrGroch** [comment]
@nicojs so i got a response there. Since there is no option like this. I'll close this PR, maybe try lurking in that project and help to do that. After we finish, I will make this PR again :)

**Comment by nicojs** [comment]
Too bad! You can still create a PR without the rule if you want, just update the variable names without the rule.  👍

**Pull request closed without merging**